Fixed UI Header bug

Gaurav-Punjabi opened the pull request:
Fixed UI Header bug by checking the type of query while adding draggable class to the header columns

Signed-off-by: Gaurav Punjabi <gaurav.bpunjabi@gmail.com>

### Description

Please describe your pull request.

Fixes #15656 

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

williamdes commented on libraries/classes/Display/Results.php:
```diff
@@ -2119,8 +2119,11 @@ private function _getDraggableClassForSortableColumns(
-        $th_class[] = 'draggable';
+        $th_class = array();
+        // The columns should not be draggable for show queries
+        if (!$this->__get('is_show')) {
```
I am unsure if this should be used as a condition, can you let me know how you did determine it was the right one ?

Gaurav-Punjabi replied:
@williamdes  Well, I tried did some debugging and came to know that 'is_show' indicates that the current query that is getting displayed is a show query. Hence we'll add class 'draggable' to only those queries that are other than show.

williamdes replied:
Okay :)

Gaurav-Punjabi committed:
Fix #15656 - UI Header bug

Fixed UI Header bug by checking the type of query while adding draggable class to the header columns

Signed-off-by: Gaurav Punjabi <gaurav.bpunjabi@gmail.com>

williamdes commented:
@Gaurav-Punjabi can you please fetch and `git reset --hard ca45710430e5a0e0cca294b6eec60b720c71108b` on this branch
While wanting to merge your fix it did not work using `SHOW PROCESSLIST`

Gaurav-Punjabi commented:
> @Gaurav-Punjabi can you please fetch and `git reset --hard ca45710430e5a0e0cca294b6eec60b720c71108b` on this branch
> While wanting to merge your fix it did not work using `SHOW PROCESSLIST`
Done I hard reset to the given has. What should I do next ?

williamdes commented:
> Done I hard reset to the given has. What should I do next ?

You you check your fix ?

See this screenshot:
![image](https://user-images.githubusercontent.com/7784660/72224857-397f8a80-357f-11ea-9ae3-67e42b0b41bc.png)

Gaurav-Punjabi commented:
> > Done I hard reset to the given has. What should I do next ?
> 
> You you check your fix ?
> 
> See this screenshot:
> ![image](https://user-images.githubusercontent.com/7784660/72224857-397f8a80-357f-11ea-9ae3-67e42b0b41bc.png)

Done 👍

williamdes commented on libraries/classes/Display/Results.php:
```diff
@@ -527,8 +527,9 @@ private function _setDisplayPartsForShow(array $displayParts)
+
         // Other settings
-        $displayParts['sort_lnk']  = (string) '0';
+        $displayParts['sort_lnk']  = (string) '1';
```
@Gaurav-Punjabi can you explain this change ?

williamdes replied:
![image](https://user-images.githubusercontent.com/7784660/72564921-191e3b80-38b1-11ea-93dd-5bc1bd6f1fd7.png)

williamdes commented:
@Gaurav-Punjabi can you please fix this PR ?

Gaurav-Punjabi commented:
@williamdes Sure I'll work on this and fix it asap 👍.

williamdes commented:
@Gaurav-Punjabi did you delete your PRs because of our very bad review delay ?
